services/alertmanager/alert.go

```diff
@@ -60,78 +47,9 @@ func (ap *AlertParams) validate() error {
 		return errors.New("empty Description")
 	}
 
-	if ap.Severity == "" {
-		ap.Severity = Info
+	if ap.Severity < check.Emergency || ap.Severity > check.Debug {
```

Comment:
I think we should verify that only when we check results, so we don't have this check in two places.

Reply:
In fact that function is unused at all.

Reply:
The whole file is not used: 59fe895223e541a6e82a4e9d9aef469a550bd815